### PR TITLE
Add -v option to print debug info

### PR DESCRIPTION
**Pull request by jkho**

### CODE REVIEW

**Comment by APriestman** [comment]
Please address the codacy complaints

File: tools/logicalimager/tsk_logical_imager.cpp [hunk 1]
```diff
@@ -917,8 +914,10 @@ main(int argc, char **argv1)
 
     if (iFlagUsed) {
         imgPaths.push_back(imgPath);
+        printDebug("Process is not running in elevated mode", NULL);
```
Comment (APriestman) [inline tools/logicalimager/tsk_logical_imager.cpp]: Is this statement accurate? Using -i seems unrelated to admin mode

File: tools/logicalimager/tsk_logical_imager.cpp [hunk 2]
```diff
@@ -946,12 +945,13 @@ main(int argc, char **argv1)
...
         const TSK_TCHAR *image = (TSK_TCHAR *)imgPaths[i].c_str();
         driveToProcess = iFlagUsed ? TskHelper::toNarrow(imgPaths[i]) : TskHelper::toNarrow(drivesToProcess[i]);
+        printDebug("Processing drive %s", driveToProcess.c_str());
 
         std::string outputFileName = directoryPath + "/" + (iFlagUsed ? "sparse_image" : driveToProcess) + ".vhd";
         std::wstring outputFileNameW = TskHelper::toWide(outputFileName);
 
         if (hasTskLogicalImager(image)) {
-            printDebug("Skipping drive %s\n", driveToProcess.c_str());
+            printDebug("Skipping drive %s", driveToProcess.c_str());
```
Comment (APriestman) [inline tools/logicalimager/tsk_logical_imager.cpp]: It would be nice to print the reason the drive is being skipped

File: tools/logicalimager/tsk_logical_imager.cpp (hunk 1 above)
Comment (jkho) [reply tools/logicalimager/tsk_logical_imager.cpp]: I will remove it.

**Comment by APriestman** [comment]
Please fix the codacy issues
